### ISSUE TITLE
Acquiring and releasing maps

### ISSUE THREAD

**Issue by Arsakes** [mapbox/mapnik-pool#9]
Could you put an example on how to use pool?
Does one have to call release explicitely after map.render?

**Comment by tmcw** [comment]
Hi @Arsakes,

Thanks for the question. Have you [seen the example in the readme](https://github.com/mapbox/mapnik-pool#example)?

- Tom

**Comment by Arsakes** [comment]
@tmcw 
Yes I did, homwever, shouldn't this exmaple contain calling for:

` pool.release(map) 

after rendering is done?
